crates/server/src/lib.rs

```diff
@@ -34,6 +58,7 @@ use log::trace;
 use tokio_util::compat::FuturesAsyncReadCompatExt;
 use tower::MakeService;
 
+#[allow(missing_debug_implementations)] // TlsAcceptor does not implement Debug
```

Comment:
A missing Debug impl means that we can't derive Debug, but we can still impl Debug manually. Do you want to do that instead?

Reply:
Not really, because there's no useful information to display.
We can print string `"App"` if you think that's useful